src/ResourceManagement/Logic/Logic.Tests/ScenarioTests/IntegrationAccountSessions.ScenarioTests.cs

```diff
@@ -0,0 +1,382 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Test.Azure.Management.Logic
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Management.Logic;
+    using Microsoft.Azure.Management.Logic.Models;
+    using Microsoft.Rest.Azure;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using Xunit;
+
+    /// <summary>
+    /// Scenario tests for the integration accounts session.
+    /// </summary>
+    [Collection("IntegrationAccountPartnerScenarioTests")]
+    public class IntegrationAccountSessionScenarioTests : ScenarioTestsBase
+    {
+        /// <summary>
+        /// Name of the test class
+        /// </summary>
+        private const string TestClass = "Test.Azure.Management.Logic.IntegrationAccountSessionScenarioTests";
```

Comment:
I will use this.GetType().FullName in the common base class to fix all existing 13 test classes.